### PR TITLE
官方已弃用 Variable

### PR DESCRIPTION
**Pull request by professordeng**

### CODE REVIEW

File: content/decision_tree/flatMap.md [hunk 1]
```diff
@@ -16,15 +16,15 @@
 let disposeBag = DisposeBag()
 let first = BehaviorSubject(value: "👦🏻")
 let second = BehaviorSubject(value: "🅰️")
-let variable = Variable(first)
+let variable = BehaviorSubject(value: first)
```
Comment (beeth0ven) [inline content/decision_tree/flatMap.md]: Also, It's better to rename `let variable` to `let subject`.

File: content/decision_tree/flatMap.md [hunk 2]
```diff
@@ -16,15 +16,15 @@
...
 
 variable.asObservable()
         .flatMap { $0 }
         .subscribe(onNext: { print($0) })
         .disposed(by: disposeBag)
 
 first.onNext("🐱")
-variable.value = second
+variable.onNext(second)
```
Comment (beeth0ven) [inline content/decision_tree/flatMap.md]: Same here.

**Review by beeth0ven — CHANGES_REQUESTED** [review]

**Review by professordeng — COMMENTED** [review]
谢谢指正，已修改～

**Comment by beeth0ven** [comment]
Thank you!

**Pull request merged**